GamepadHapticsActuatorTriggerRumble/explainer.md

```diff
@@ -96,9 +97,11 @@ Allowing websites to query for Gamepads' haptic capabilities might raise fingerp
 
 ## Alternative Solutions
 
-### `canPlay` versus a new `GamepadHapticActuatorType` entry
+### `GamepadHapticActuatorType` entry for effect support detection
+Adding a new entry entry, e.g. "trigger-rumble" to GamepadHapticActuatorType might be difficult to accomplish without breaking existing apps. For example, an app that always checks for "dual-rumble" may break if Xbox controllers suddenly begin to return "trigger rumble" as their `GamepadHapticActuatorType`. Therefore, it may be better to let the web applications do the required capability detection using a different route - e.g., by calling a `canPlay` method -; and always define `GamepadHapticActuator.type` to be "dual-rumble", so apps cannot rely on it, then deprecate, and eventually remove it.
```

Comment:
nit: Should we update the example to use effects instead of canPlay?

Reply:
Sorry. Forgot about it.